src/app/app-wrapper.js

```diff
@@ -1,22 +1,21 @@
 import { CssReset, CssVariables } from '@dhis2/ui'
 import React from 'react'
-import { QueryClientProvider } from 'react-query'
+import { ReactQueryDevtools } from 'react-query/dist/react-query-devtools.development'
```

Comment:
Not sure -- I piggy-backed off @Birkbjo's code here, but I'll try out that import 

That said, this is a temporary addition, so a wonky import is probably acceptable for now

Reply:
But nice, looks like the v4 full release is out since this PR was started 🙂 guess it's time to fully migrate?

Reply:
Yeah this was something wacky with the beta. I haven't tried the most recent version yet, but for some reason the build failed when trying to import it from `react-query/devtools`. 
But yeah this will probably be changed and fixed in the stable v4 version.